pdns/dnsdistdist/dnsdist-web.cc

```diff
@@ -515,8 +516,11 @@ static void handlePrometheus(const YaHTTP::Request& req, YaHTTP::Response& resp)
       // for these we have the help and types encoded in the sources
       // but we need to be careful about labels in custom metrics
       std::string helpName = prometheusMetricName.substr(0, prometheusMetricName.find('{'));
-      output << "# HELP " << helpName << " " << metricDetails.description << "\n";
-      output << "# TYPE " << helpName << " " << prometheusTypeName << "\n";
+      if (!helpAndTypeSent.count(helpName)) {
```

Comment:
clang-tidy does no like the implict int->bool conversion
```suggestion
      if (helpAndTypeSent.count(helpName) == 0) {
```

Reply:
Oh, right, of course!